### PR TITLE
Improved readme and sample script of energy forecast api.

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,22 +320,41 @@ the corresponding string in the `fee_type` input parameter.
...
 POST https://d3aweb-dev.gridsingularity.com/api-token-auth/ 
-Form Body {username: <your_username>, password: "your_password"}
-# Send the JWT token via the Authorization HTTP header when sending the measurement data 
-# (add "Authorization: JWT <your_token>" to the HTTP headers)
+```
+The body of the request needs to contain the following information (JSON string):
+```
+{"username": "<your_username>", "password": "<your_password>"}
+```
+The returned JWT needs to be sent via the Authorization HTTP header when sending the forecast data
```
[the remark this answers is not in the document]
Comment (fievelk) [reply README.md:347]: (The output markdown looks weird without this dot)

File: README.md [hunk 2]
```diff
@@ -320,22 +320,41 @@ the corresponding string in the `fee_type` input parameter.
...
+```
+{"username": "<your_username>", "password": "<your_password>"}
+```
+The returned JWT needs to be sent via the Authorization HTTP header when sending the forecast data 
+For that you need to add the following key value pair to the header of the POST command:
+```
+Authorization: JWT <your_token>
+```
+###### Send energy forecast
+The POST  to send the energy value is (please fill in `<Canary Network UUID>` and `<Device UUID>`:
```
Comment (fievelk) [inline README.md:353]:
```suggestion
The POST to send the energy value is the following (please fill in `<Canary Network UUID>` and `<Device UUID>`):
```

File: d3a_api_client/setups/test_sending_energy_forecast.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.current_pv_energy_forecast_Wh = 0
+        self.current_load_energy_forecast_Wh = 0
+
+    def on_market_cycle(self, market_info):
+        """Perform following code on when a new market was started."""
+        # set energy forecasts for the next market for load and pv
+        load.set_energy_forecast(self.current_pv_energy_forecast_Wh)
+        pv.set_energy_forecast(self.current_load_energy_forecast_Wh)
```
[the remark this answers is not in the document]
Comment (hannesdiedrich) [reply d3a_api_client/setups/test_sending_energy_forecast.py:21]: Haha, of course, thanks!

File: README.md (hunk 2 above)
Comment (fievelk) [reply README.md:353]: There was a missing parenthesis mostly

File: d3a_api_client/setups/test_sending_energy_forecast.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+class TestAggregator(Aggregator):
+    """Aggregator used for receiving events from CN"""
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.current_pv_energy_forecast_Wh = 0
+        self.current_load_energy_forecast_Wh = 0
+
+    def on_market_cycle(self, market_info):
+        """Perform following code on when a new market was started."""
```
Comment (fievelk) [inline d3a_api_client/setups/test_sending_energy_forecast.py:18]: Nit:
```suggestion
        """Perform following code when a new market is started."""
```

**Review by fievelk — COMMENTED** [review]
Some minor comments and a slightly more important one

**Review by chrisd12 — APPROVED** [review]
LGTM

**Review by fievelk — APPROVED** [review]

**Pull request merged**